spec/features/merchant_site/item/show_spec.rb

```diff
@@ -41,14 +41,18 @@
   it 'merchant item show page lists name, description, and selling price' do
     # As a merchant, when I click on the name of an item from the merchant items index page, (merchants/:merchant_id/items)
     visit merchant_items_path(@merchant)
-    click_link "#{@item_list[0].name}"
+    save_and_open_page
+    within ".top_5_items" do
+      click_link @item_list[0].name      
+    end
```

Comment:
This is where there are sometimes ambiguities in bundle exec rspec - we will address this together